include/deal.II/particles/particle_handler.h

```diff
@@ -641,7 +641,7 @@ namespace Particles
      * triangulation.
      */
     types::particle_index
-    n_locally_owned_particles() const;
+    n_locally_owned_particle_ids() const;
```

Comment:
I know I am late to this PR, sorry for that. Why was this function name changed as well? This is not really part of the index discussion, and was a function that was in here for some years. This renaming also makes it inconsistent with the `n_global_particles()` function above. Unless there is a specific reason for this change, I would prefer if we could revert this particular change (only this function name). (Also see the problems reported in https://github.com/geodynamics/aspect/issues/4023#issuecomment-832036067).

Reply:
This was an accident. Sorry. PR is coming!

Reply:
No harm done, thanks for the quick reply!

Reply:
Oh, and I totally missed it too :-(

Reply:
Thanks @gassmoeller . Good catch :).